src/main/resources/application.properties

```diff
@@ -12,8 +12,8 @@ spring.jpa.properties.hibernate.format_sql: false
 app.sendgrid.key:
 app.sendgrid.from-address:
 #JWT
-jwt.secret =${SECRET} 
-jwt.authorization ={AUTHORIZATION}
+jwt.secret =${SECRET}
```

Comment:
There is no need for this change, with Lucas merged PR is solved

Reply:
Update local branch with lastest version of main.